### ISSUE TITLE
Text to speech - Does not have a stop reading option

### ISSUE THREAD

**Issue by R-R-M** [joomla/joomla-cms#30690]
### Steps to reproduce the issue
Enable accessibility component. Click text to speech then click an element on the front end to read out aloud. It is not possible to stop it reading the element out aloud. Is this the bug or a lack of functionality?

### Expected result
The user should be able to stop the text to speech by clicking on another element that they want to hear read out or by clicking the same element to stop it reading. It would also be better if the element that it was reading was more obvious. Perhaps a red dashed border around the current reading element. The slight drop shadow is not visually nice or obvious.

**Comment by chmst** [comment]
A11y issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30690">issues.joomla.org/tracker/joomla-cms/30690</a>.</sub>

**Comment by charvimehradu** [comment]
By element do you mean any other accessibility options like for example if we click on big cursor it should not read "big cursor" and should automatically disable the function fully?

**Comment by fgsw** [comment]
Firefox 136.0.2, macOS 14.7.4, Joomla 5.2.5: In the Accessibility Checker is no ```text to speech``` as @R-R-M mentioned.
